bundles/org.openhab.binding.solax/src/main/resources/OH-INF/thing/localConnectEVCharger.xml

```diff
@@ -0,0 +1,135 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="solax"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<thing-type id="local-connect-charger">
+
+		<label>Local Connect Charger</label>
+		<description>The charger representation that supports local connections via HTTP</description>
+
+		<channels>
+			<channel id="eq-single-session" typeId="system.electric-energy">
+				<label>Energy charged this session</label>
```

Comment:
Please check all labels:  23 < characters, 2-3 words and capitalized.

As all channels are part of the charger, you can probably already remove all the Charger prefixes and Phase 1, Phase 2 can be shortened to P1, P2 etc. Just some suggestions.